src/fhir-tool-core/Operations/GenerateBinaryOperation.cs

```diff
@@ -0,0 +1,89 @@
+﻿extern alias R4;
+using CommandLine;
+using FhirTool.Core.ArgumentHelpers;
+using R4::Hl7.Fhir.Model;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+using FhirTool.Core.FhirWrappers;
+using System.IO;
+using System.Linq;
+
+namespace FhirTool.Core.Operations
+{
+    [Verb("generate-binary", HelpText = "Generates binary file")]
+    public class GenerateBinaryOperationOptions
+    {       
+        [Option('i', "id", HelpText = "Binary id", Required = true)]
+        public string Id { get; set; }
+
+        [Option('c', "contentType", HelpText = "Content type", Required = true)]
+        public string ContentType { get; set; }
+
+        [Option('s', "securityContext", HelpText = "Security context", Required = true)]
+        public string SecurityContext { get; set; }
+
+        [Option('f', "file", HelpText = "file", Required = true)]
+        public WithFile File { get; set; }
+
+        [Option("fhir-version", MetaValue = "fhirVersion", HelpText = "fhir version")]
+        public FhirVersion FhirVersion { get; set; }
+
+        [Option('m', "format", MetaValue = "xml/json", HelpText = "json or xml")]
+        public FhirMimeType MimeType { get; set; }
+
+        [Option('p', HelpText = "Save binary to path", Required = true)]
```

Comment:
Endre short argument til 'o' og argument til 'out': `[Option('o', "out", HelpText = "Out path for Binary", Required = true)]` og navn på property tilsvarende `OutPath`